etl_pipeline/extract/base_extractor.py

```
@@ -41,21 +43,40 @@ def validate_config(self) -> bool:
+                
+                # Check that the path actually exists
+                path_str = str(path_obj)
+                # Detect URLs by known schemes or "://" pattern
+                is_url = FileUtils.is_url(path_str)
+                # Only check local filesystem paths
+                if not is_url:
+                    if not path_obj.exists():
+                        logger.error(f"Configuration path does not exist: {path_obj.absolute()}")
+                        return False
+                
+                return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)